packages/wrangler/src/dev/dev.tsx

```diff
@@ -466,11 +557,11 @@ function DevSession(props: DevSessionProps) {
 			);
 		}
 
-		if (bundle.current) {
+		if (bundle) {
 			latestReloadCompleteEvent.current = {
 				type: "reloadComplete",
 				config: startDevWorkerOptions,
-				bundle: bundle.current,
+				bundle: bundle,
```

Comment:
```suggestion
				bundle,
```

Reply:
Besides consistency, there's no downside to this syntax. The lint rule will be more annoying than beneficial imo

Reply:
Arguably one of the main points of linting _is_ consistency though

Reply:
I've fixed this for now—let's revisit our linting/prettier rules here in a followup